README.md

```diff
@@ -1,4 +1,23 @@
 # icgc-download-client
 Universal download client for ICGC data residing in various environments. 
 
-*WORK IN PROGRESS*
+## Installing the python script:
+
+The python script can be installed by simply navigating to the icgc-download-client directory and running the command
+"python setup.py install"
+
+## Using the python script:
+
+The required arguments for the python script are the repository that is being targeted for download.
+Valid repositories are aws _(Amazon Web Services)_, collab _(Collabratory)_, ega _(European Genome Association)_
+gdc _(Genomic data commons)_ and cghub _(Cancer genomic hub)_
+
+Second you must specify either a file id using the tag -f or --file_id, a manifest file using the tags -m or -manifest
```

Comment:
Use backticks for literal arguments, such as `-f` and `--file_id`.

Btw, it should be `--file-id` and not `--file_id` as this is the standard convention.